### DISCUSSION TITLE
When transferring files how can I access my external storage drive (sd card)?

### DISCUSSION THREAD

**Discussion by Savasdotexe** [rustdesk/rustdesk#6154] [Q&A]
Thought to ask, I must be missing it, but I can't seem to see how to switch from the internal storage to external sd card?

I've looked and can't see any similar questions so I am guessing this is not a feature request....

**Comment by My1** [comment]
maybe it might be useful to clarify what OSes are even involved

**Reply by Savasdotexe** [reply]
Android device. So transferring files between a Windows 10 and Android 13.
